Yoast/wpseo-woocommerce
# Remove distribution files from repository

## Summary

This PR can be summarized in the following changelog entry:

* None needed.

## Relevant technical choices:

*  Moved the JS files to a `dist` directory within the `js` directory.
* Updated plugin-grunt-tasks to the last minor version in the same go.

## Test instructions

This PR can be tested by following these steps:

* Make sure the right files are still loaded on product edit screens.

Fixes #240
